event_registration_cancel_reason/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================================
+Reasons for event registrations cancellations
+=============================================
+
+This module allows to add a predefined reason why a registration for an event
```

Comment:
This module allows to add predefined reasons for the attendees to report back why they cancel their registration to an event.